crates/tests/winrt/tests/object.rs

```diff
@@ -3,7 +3,7 @@ use windows::Foundation::{IStringable, PropertyValue, Uri};
 
 #[test]
 fn class() -> windows::core::Result<()> {
-    let uri = Uri::CreateUri("http://kennykerr.ca")?;
+    let uri = Uri::CreateUri(&windows::core::HSTRING::from("http://kennykerr.ca"))?;
```

Comment:
Perhaps we just need to have a universal PCWSTR type for all wide string input parameters to avoid the complication of dealing with the various different types. Then we can specialize that one to handle all the marshaling under the hood - that should simplify and harmonize the various calling patterns for strings. Not for this PR but need to land that fairly closely along with this.